ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -593,9 +593,8 @@ presubmits:
     context: ci/prow/e2e-libvirt
     decorate: true
     name: pull-ci-openshift-installer-master-e2e-libvirt
-    optional: true
```

Comment:
But @praveenkumar informed you that is not true anymore. Was he wrong?

Reply:
@abhinavdahiya we are collecting the must gather logs, what logs should we collect in case of bootstrap failure, we can add it also. My assumption was that must gather logs should have all the details around cluster.

Reply:
During bootstrap failure, the api is not available, neither is the cluster to run must-gather.. so bootstrapping failed errors is not debugable with must-gather.

the installer run the equivalent of `openshift-install gather bootstrap` when bootstrapping fails to gather the logs from the bootstrap-host using SSH.

Reply:
@abhinavdahiya thanks we will experiment with `openshift-install gather bootstrap` and add it to the libvirt template.

Reply:
@abhinavdahiya After https://github.com/openshift/installer/pull/2120 is merged (it's queued), we'll also have the bootstrap logs now. So I think these changes should be good to go now.

Reply:
openshift/installer#2120 is merged.